### ISSUE TITLE
Custom e-Tables (drop lists)

### ISSUE THREAD

**Issue by fishbaugher** [fishbaugher/WAweb-Issues#2]
The desktop version of WA allowed states to modify all drop lists including measure types.  If ORNL allows the drop lists to be edited, then the primary list of measure types could be expanded to include all the sub types since we maintained unique ID codes for all measure type and measure sub type selections in v8.11, the same applies to cost center type and sub type selectors.  In ORNL language this would be "user editable e-table contents" (MJF)

Here are the custom 8.11 drop lists for cost center type and work order measure types.  Both are cascading (new in 8.11 for MN) but 8.x has always allowed custom drop list contents via a backend table named tblzSelector which contains the drop list contents and identifiers for the entire user interface.  Here are the screen shots of the cascading selectors in WA 8.11
<img width="564" alt="Untitled" src="https://github.com/fishbaugher/WAweb-Issues/assets/479175/85283612-86bd-45c8-b324-8dbb880437c3">
<img width="595" alt="Untitled 2" src="https://github.com/fishbaugher/WAweb-Issues/assets/479175/3b1cad3a-b818-47c2-aea9-1926ef5f7362">
<img width="713" alt="Untitled 3" src="https://github.com/fishbaugher/WAweb-Issues/assets/479175/e43295fb-da37-4053-9547-5e6fe173bd4a">
<img width="715" alt="Untitled 4" src="https://github.com/fishbaugher/WAweb-Issues/assets/479175/c2252022-a2c6-4cd9-84ae-6e3fcedc2c35">

And here are the same selectors in WAweb.

<img width="514" alt="Untitled5" src="https://github.com/fishbaugher/WAweb-Issues/assets/479175/b8846a50-f3a9-4512-bc5f-c8deabc9d97b">
<img width="435" alt="Untitled6" src="https://github.com/fishbaugher/WAweb-Issues/assets/479175/d7b2a1ae-75cd-43ca-8daf-158e770ecc84">

Without the ability to characterize the cost center or work order measure grouping (type) in a systematic way, we would have to go back to relying on naming conventions for cost centers and measures which are problematic in practice.

**Comment by fishbaugher** [comment]
This is a feature in 8.12 not 8.6, so it is really a new feature only request.

**Comment by fishbaugher** [comment]
In the absence of cascading drop lists, I propose to standardize on the cost center naming convention as shown in the attached spreadsheet.

[CostCenterNamingConvention.xls](https://github.com/fishbaugher/WAweb-Issues/files/14729001/CostCenterNamingConvention.xls)

This file also added to the top level path in this repository code.

**Comment by eckmanwe** [comment]
To clarify/restate. The request is to modify the current Cost Center Type enumeration nationwide based upon MN's implementation. [Please Confirm]  

Similarly, the requested list is as follows [Please Confirm]:

US DOE
State
Utility
Donated
Other Federal
Other funds   included in grant budget
Other funds   not included in grant budget
LIHEAP (see   State/EAP) (inactive)
Program
Carryover   (inactive)
BIL
Wx Readiness   (WRF)
EI WIRE
SERC/WRAP
SERC-2
CSPP
ARRA   (inactive)
Other
EAP/WX Program
EAP/WX   Carryover
Healthy AIR   State
Propane
Oil (inactive)
EAP/WX ARPA   (inactive)
Other
Healthy AIR   CIP
MERC   (inactive)
Other

**Comment by fishbaugher** [comment]
No, the request it to open up the e-tables for each state so they can create state specific super-set lists that comply with the national list, but allow for extension.

Funding sources and measure types are two of the things that seem to change each program year.  Perhaps if those drop lists were kept current (yea _hard_)with new funding sources and measure rules, then there would be no need to create custom lists.

In MN for v8.12 we implemented nested two-level selectors for both cost centers and work order measure types.  A fair bit of thinking went into the list and portions of that work might be applicable nationally.

But, for now, some way to expand the lists like available in 8.9 would be wonderful.  In v8.9 the tblzSelector was stored in the _backend_ MDB file, so the user could adjust the lists just by editing that one backend table...
